js/classes/entities/foreground/dialog/conversation.js

```diff
@@ -5,12 +5,12 @@
  */
 class Conversation {
     /**
-     * @param {Array<DialogBubble>} array The array containing all possible dialog bubbles in this conversation.
+     * @param {Array<DialogBubble|Choice>} array The array containing all possible dialog bubbles in this conversation.
```

Comment:
Good design choice to accept both DialogBubbles and DecisionBubbles. Just a note from our conversation to remember to change `Choice` to `DialogBubble`.